src/screens/UserManagement/UserRevamp/NewUserInvitationForm.res

```diff
@@ -195,19 +197,19 @@ let make = () => {
         // <FormValuesSpy />
       </div>
       <div className="p-6 flex flex-col gap-2 col-span-3">
-        {switch roleTypeValue {
-        | Some(role) =>
+        {switch (roleTypeValue, roleNameValue) {
```

Comment:
`roleNameValue` is not required in switch case, as it is not impacting how switch behaves in this case, you can remove this extra code changes

Reply:
Got it and Removed.